app/views/events/index.html.erb

```diff
@@ -28,6 +29,9 @@
           <td><% event.guests.each do |guest| %>
                 <%= guest.firstname %> <%= guest.lastname%>,
               <% end %>
+          <%= link_to 'http://www.facebook.com/sharer.php?u='+request.fullpath, :target => :blank do %>
```

Comment:
This didn't work when I tried it?

![image](https://user-images.githubusercontent.com/25331886/56930846-e918d900-6a92-11e9-84a9-4f0096b1255f.png)

https://www.facebook.com/sharer.php?u=/events/

Reply:
See above comment